libraries/Microsoft.Bot.Builder.LanguageGeneration/Evaluator.cs

```diff
@@ -298,6 +292,26 @@ public object ConstructScope(string inputTemplateName, List<object> args)
             return new CustomizedMemory(memory.GlobalMemory, new SimpleObjectMemory(newScope));
         }
 
+        // Validator for template(...)
```

Comment:
Please revert the code block if there is no change on it.

Reply:
reverted